browser/profiles/brave_renderer_updater.cc

```diff
@@ -79,12 +83,14 @@ BraveRendererUpdater::BraveRendererUpdater(
 #endif
 
 #if BUILDFLAG(ENABLE_WIDEVINE)
-  widevine_enabled_.Init(kWidevineEnabled, local_state);
-  local_state_change_registrar_.Init(local_state);
-  local_state_change_registrar_.Add(
-      kWidevineEnabled,
-      base::BindRepeating(&BraveRendererUpdater::UpdateAllRenderers,
-                          base::Unretained(this)));
+  if (local_state) {
```

Comment:
for consistency all of these should probably use `local_state_`. Also can we combine it into one `if/else` so it's a little easier to read/understand?

Reply:
Changed in [5ed2e32](https://github.com/brave/brave-core/pull/20433/commits/5ed2e32584c5baca74b5c55b618643e34ee6cd8f)